doc/cephfs/eviction.rst

```diff
@@ -102,7 +102,7 @@ the blacklist:
 
     ceph osd blacklist ls
     # ... identify the address of the client ...
-    ceph osd blacklist rm <address>
+    ceph osd blacklist rm <client-instance>
```

Comment:
Let's have an example with real output from `blacklist ls` and a real instance name passed to `blacklist rm`.